<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettings</class>
 <widget class="QDialog" name="ConnectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="connectionTypeComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>тип</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="comPortSpinBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipAddressLineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dataBitsComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>datdbits</string>
   </property>
  </widget>
  <widget class="QComboBox" name="stopBitsComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>stopBits</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baudRateComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentText">
    <string extracomment="dawd"/>
   </property>
   <property name="placeholderText">
    <string>baud</string>
   </property>
  </widget>
  <widget class="QComboBox" name="parityComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>parity</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
